test/unit/math/p5.Vector.js

```diff
@@ -1882,5 +1882,48 @@ suite('p5.Vector', function() {
         expect(p5.Vector.equals(a1, a2)).to.be.true;
       });
     });
+
+    suite('p5.Vector.clampToZero()', function() {
+      let v;
+
+      test('should clamp very small positive number to zero', function() {
+        v = new p5.Vector(0.0000000000000002220446049250313, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(5);
+      });
+
+      test('should clamp very small negative number to zero', function() {
+        v = new p5.Vector(-0.0000000000000002220446049250313, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(5);
+      });
+
+      test('should not clamp regular numbers', function() {
+        v = new p5.Vector(0.01, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0.01);
+        expect(v.y).to.equal(5);
+      });
+
+      test('should leave zero components unchanged', function() {
+        v = new p5.Vector(0, 0);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(0);
+      });
+
+      test('should clamp very small numbers in all components of a 3D vector to zero', function() {
```

Comment:
I like how you mentioned that we are clamping "components of a vector", I think it would be awesome if other tests followed the same wording 


ie instead of saying that we are rounding down the number we can say rounding down the component of a vector